units/systemd-vconsole-setup.service.in

```diff
@@ -11,6 +11,8 @@
 Description=Setup Virtual Console
 Documentation=man:systemd-vconsole-setup.service(8) man:vconsole.conf(5)
 DefaultDependencies=no
+Wants=systemd-udevd.service
+After=systemd-udevd.service
```

Comment:
I still don't understand the point of these deps: the fact that udevd has been started doesn't mean that the vconsole underlying device is ready to be used. In theory the kernel can send the corresponding event at any time and there's no way for a unit to be ordered against this event (unless a dedicated device unit is created).

As the vconsole setup is not mandatory for using the console, I think `s-vc-s.service` shouldn't have any impact on the scheduling of the services depending on it. Those services should just try to run `s-vc-s.service` before they're run and if the console can be setup then fine otherwise just proceed anyway.